docs/source/ops/index.rst

```diff
@@ -0,0 +1,65 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. ops-ref_index:
+
+Ledger API Server pruning
+=========================
+
```

Comment:
```suggestion
```suggestion
DAML Participant pruning is currently an :doc:`Early Access Feature in Labs status </support/status-definitions>`

```
```

Reply:
Done.